ccai/config.py

```diff
@@ -31,8 +31,8 @@ class ConfigSingleton(Singleton):
 
     SECRET_KEY = os.environ.get("SECRET_KEY") or "secret-key"
     API_KEYS_NAME = ["GEO_CODER_API_KEY", "STREET_VIEW_API_KEY"]
-    GEO_CODER_API_KEY = "AIzaSyBDueAV41YFgKvKZ72UtTHQ3PiMuedwZ0k"
```

Comment:
!! @jsnrdl what is this? How come this is committed?

Reply:
API keys should **NEVER** be committed... did you generate new ones?

Reply:
that has been modified, the keys have been changed and put in a file.